src/components/WwwFrame/TheHeader.vue

```diff
@@ -3,7 +3,7 @@
 		<nav aria-label="Primary navigation">
 			<div class="header-row row">
 				<router-link class="header-logo header-button" to="/" v-kv-track-event="['TopNav','click-Logo']">
-					<kv-icon name="new-kiva-logo" />
```

Comment:
I don't see `new-kiva-logo` being used anywhere else besides styleguide, so let's go ahead and remove that icon file too so that it doesn't get bundled into the icon sprite svg anymore

Reply:
👍 Do you know if the old stack will still use it in some way?

Reply:
The old stack uses a different sprite bundle, so it won't be affected. We could consider inlining the svg for the old stack too, actually.